### PR TITLE
Blackout footer

### PR DESCRIPTION
**Pull request by kdzapp**
Fixes # .
#39

Added Assets for Footer

### PR Description

Footer Progress

### What needs to be reviewed?

Footer Alignment and Placement is messed up and I'm not sure how to get it where it needs to be. ImgButton has to be link to webpages.

### CODE REVIEW

File: app/pages/blackout/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Container } from '../../components';
+
+const HeaderLogoImage = require('../../../static/icons/blackout-logo.png');
+
+const InstagramImg = require('../../../static/icons/Instagram.png');
```
Comment (connorkrupp) [inline app/pages/blackout/footer.jsx]: Can you make these images lowercase pl0x

File: app/pages/blackout/footer.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+const TwitterImg = require('../../../static/icons/Twitter.png');
+
+/* Footer Style */
+const Footer = styled.footer`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 1rem;
+  text-align: center;
```
Comment (connorkrupp) [inline app/pages/blackout/footer.jsx]: Camelcase property name

File: app/pages/blackout/footer.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const HeaderLogo = styled.img`
+  	display: block;
+    height: 90px;
+`;
+
+const ImgButton = styled.img`
+	margin: 10px;
+	padding: 5px;
+    height: 50px;
+    display: inline-block;
```
Comment (connorkrupp) [inline app/pages/blackout/footer.jsx]: Camelcase

File: app/pages/blackout/index.jsx [hunk 4]
```diff
@@ -1,41 +1 @@
...
            </Page>
        );
    }
}

function mapStateToProps(/*state*/) {
    return {};
}

export default connect(mapStateToProps)(BlackoutPage);
```
Comment (connorkrupp) [inline app/pages/blackout/index.jsx]: Woe what happened here

Comment (roball24) [inline app/pages/blackout/index.jsx]: the file looks fine on my machine. looks like a github problem

**Pull request closed without merging**